src/turbo-stream.spec.ts

```diff
@@ -50,6 +50,13 @@ test("should encode and decode Date", async () => {
   expect(output).toEqual(input);
 });
 
+test("should encode and decode Invalid Date", async () => {
+  const input = new Date('invalid');
+  const output = await quickDecode(encode(input)) as Date;
+  expect(output?.toString()).toEqual('Invalid Date');
+  expect(output?.getTime()).toBe(NaN);
```

Comment:
This is unusual, because jest's expect matchers don't work properly with Invalid Dates.